src/f-utils/index.js

```diff
@@ -0,0 +1,77 @@
+'use babel'
+
+import r from '../f-utils/mini-ramda'
+
+// random comment to test out `hub` behavior
+
+/** *************
+ * Typecasting *
+ ************** */
+
+// fromMapToObj :: Map a -> Object a
+export const fromMapToObj = m => r.fromPairs([...m])
+
+// fromObjToMap :: Object a -> Map a
+export const fromObjToMap = o => new Map(r.toPairs(o))
+
+
+/** ************
+ * Predicates *
+ ************* */
+
+// False if parameter is null or undefined, else true
+//
+// isNotNil :: Any -> Boolean
+export const isNotNil = r.pipe(r.isNil, r.not)
```

Comment:
I was going to say this JavaScript not LUA... but that's the term Ramda uses for some bizarre reason 😮.

Reply:
I think it's nice to have a term that refers to both undefined and null, since they are so often used interchangeably.